src/06_tuples.py

```diff
@@ -36,9 +36,13 @@ def dist(a, b):
 
 # YOUR CODE HERE
 
+def print_tuple(a):
+    for i in range(0, len(a)):
```

Comment:
if the start of your range is "0" you can optionally leave it off and it'll still work the same way